src/helper/CONSTANT.ts

```diff
@@ -0,0 +1,3 @@
+export enum CONSTANT  {
+    StatusBroadcast = "status@broadcast"
+};
```

Comment:
I'd move this to the `config.ts` file as well and use it as const for better intellesence. -1 file

```suggestion
export const constants =  {
    StatusBroadcast = "status@broadcast"
} as const;
```